frontend/src/metabase/nav/containers/AppBar/AppBar.unit.spec.tsx

```diff
@@ -0,0 +1,200 @@
+import React from "react";
+import nock from "nock";
+import { screen, waitFor } from "@testing-library/react";
+import {
+  createMockCard,
+  createMockCollection,
+  createMockUnsavedCard,
+} from "metabase-types/api/mocks";
+import { renderWithProviders } from "__support__/ui";
+import {
+  createMockAppState,
+  createMockEmbedState,
+  createMockQueryBuilderState,
+} from "metabase-types/store/mocks";
+import { setupCollectionsEndpoints } from "__support__/server-mocks";
+import { EmbedOptions } from "metabase-types/store";
+import AppBar from "./AppBar";
+
+describe("AppBar", () => {
+  const matchMediaSpy = jest.spyOn(window, "matchMedia");
+
+  describe("full-app embedding", () => {
```

Comment:
Almost all of the tests are ported from `embedding-full-app.cy.spec.js`. Only tests that can't be unit tested are left there.